lib/requests.js

```
@@ -56,8 +56,10 @@ module.exports = {
 
     this.settings = _.extend(this.settings, settings);
 
+    if (this.isZatEnabled()) return {};
```

why not just let it fail?